### ISSUE TITLE
Update to .NET Core 2.2

### ISSUE THREAD

**Issue by HeyJoel** [cofoundry-cms/cofoundry#270]
It's not released yet, but we can use this issue to log any potential work:

- [ ] `StaticFilePathFormatter`:We use `FileVersionProvider` which is in an *Internal* AspNetCore namespace and looking at #269 it would seem it has been moved or changed in some way.

**Comment by HeyJoel** [comment]
Fixed, will be released in v0.6.

**Comment by HeyJoel** [comment]
The verb route constraint in APIs is not respected if the compatibility version flag is set to 2.2 so I've reverted it for now. I'm reopening this and logging it here:

Routes are registered using `AdminApiRouteBuilderContext` and the exception that gets thrown is `AmbiguousMatchException` between two paths with different verbs. Regustration code:

```
_routeBuilder.MapRoute(
                $"Cofoundry Admin Module - {verb}: {path}",
                _adminSettings.DirectoryName + "/api/" + path,
                new { controller = controllerName, action = actionName, Area = RouteConstants.AdminAreaName },
                constraints: new RouteValueDictionary(new { httpMethod = new HttpMethodRouteConstraint(verb) })
                );
```
